.github/workflows/ruby_ci.yml

```diff
@@ -0,0 +1,45 @@
+# .github/workflows/ruby_tests.yml
```

Comment:
?

Reply:
Ahh, began with the basic template file in https://github.com/JamesGlover/travis_to_gh_actions Eating my own dogfood an all. I'll remove that.